tests/integration/test_data/test_restore_v2.log

```diff
@@ -1,3 +1,4 @@
 /V2
 7B0A20202020227375626A656374223A20227375626A6563742D31222C0A202020202276657273696F6E223A20312C0A20202020226D61676963223A20312C0A20202020226B657974797065223A2022534348454D41220A7D	7B0A202020202264656C65746564223A2066616C73652C0A20202020226964223A20312C0A2020202022736368656D61223A20225C22737472696E675C22222C0A20202020227375626A656374223A20227375626A6563742D31222C0A202020202276657273696F6E223A20310A7D
 7B0A20202020227375626A656374223A20227375626A6563742D31222C0A202020202276657273696F6E223A20322C0A20202020226D61676963223A20312C0A20202020226B657974797065223A2022534348454D41220A7D	7B0A202020202264656C65746564223A2066616C73652C0A20202020226964223A20322C0A2020202022736368656D61223A20225C22737472696E675C22222C0A20202020227375626A656374223A20227375626A6563742D31222C0A202020202276657273696F6E223A20320A7D
+7B0A20202020227375626A656374223A20227375626A6563742D31222C0A202020202276657273696F6E223A20322C0A20202020226D61676963223A20312C0A20202020226B657974797065223A2022534348454D41220A7D	
```

Comment:
Value is null.
Schema topic backup has to have key, so that case is not presented in this data.